app/experimenter/nimbus-ui/templates/nimbus/index.html

```diff
@@ -0,0 +1,28 @@
+{% load static %}
+
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="utf-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <link rel="icon" href="{% static "imgs/favicon.ico" %}" />
+  <meta name="description" content="" />
+  <meta name="referrer" content="origin" />
+  <meta name="robots" content="noindex,nofollow" />
+  <meta name="viewport"
+    content="width=device-width,initial-scale=1,minimum-scale=1,maximum-scale=2,user-scalable=yes" />
+  <title>Mozilla Experimenter</title>
+</head>
+
+<body>
+  <noscript>Mozilla Experimenter requires JavaScript to run.</noscript>
+  <div id="root" data-config="{{ APP_CONFIG }}"></div>
+
+  {% if not DEBUG %}
```

Comment:
@3670 sorry wait, shouldn't this be the other way around? if `DEBUG` we DO want to use localhost?

Reply:
Damn, yeah, sorry that's a relic of testing the prod build. Fixed.